### ISSUE TITLE
Detect Music Start, not Sound Start

### ISSUE THREAD

**Issue by daschuer** [mixxxdj/mixxx#10885]
### Feature Description

Some tracks are starting with an useless ambient sound like: 
[Oasis - Champagne Supernova](https://music.youtube.com/watch?v=NcUhYQQrvyI)

Our first sound detection with a fixed -60 dB threshold works reliable to detect the first sound, however it picks the ambient sound start. For Auto DJ, we may want the Music Start. In the case above, the threshold that is not triggered by the ambient noise is at -12 dB. Unfortunately the Cue is than set a bit late, because is misses the build up of the actual music. So we may use something more than just a simple threshold.

**Comment by ChasonDeshotel** [comment]
suggestion: detect the downbeat where the music starts then set the first cue to the closest-to-the-start multiple of 32 bars. Maybe a configurable max of -32, -64, -96 bars? I don't know how the metadata works but if we could store how many bars of ambient intro a song has you could skip part for auto DJ but still be able to cue up normally when mixing

It would be nice to do something with the rest of the song to detect the downbeat, as some songs have ambient intros and then don't start on the one. This usually tricks Traktor into shifting the grid... idk if that means it's difficult to detect, resource intensive, or if they just didn't bother to fix it

**Comment by daschuer** [comment]
Yes that makes sense. A prereqesite would be a downbeat detection. We have some ideas, but the work has stalled. From the first reliable downbeat we can go backwards to a position where the not detectable first beat is. For some tracks, it us even before the track start. 
Related:
https://github.com/mixxxdj/mixxx/issues/10164
https://github.com/mixxxdj/mixxx/issues/9676
https://github.com/mixxxdj/mixxx/issues/8760
